scripts/west_commands/runners/bossac.py

```diff
@@ -51,14 +54,27 @@ def do_run(self, command, **kwargs):
             raise NotImplementedError(msg)
 
         self.require(self.bossac)
-
         if platform.system() == 'Linux':
             self.require('stty')
             cmd_stty = ['stty', '-F', self.port, 'raw', 'ispeed', '1200',
                         'ospeed', '1200', 'cs8', '-cstopb', 'ignpar', 'eol', '255',
                         'eof', '255']
             self.check_call(cmd_stty)
 
+            # Wait until target reboots
+            sleep(0.250)
+            # Wait until port re-enumerates
+            retries = 0
+            while (retries < 3):
```

Comment:
`while retries < 3:` since this is Python, not C

Reply:
Ah yes. That was a silly one.